portfolio/src/main/webapp/index.html

```diff
@@ -7,16 +7,55 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is the Moment.</p>
-      <p> My name is Obi Nnaeto and I am a sophomore Informations systems
-          Major at Carnegie Mellon University. I am from Nigeria and I live in New Jersey.
-      </p>
-      <p>Click here to get a quote from my fav tv shows :) :</p>
-      <img src="images/grad_pic.jpeg"/>
+    <div id= "content">
+     <div id = "top_row">
+        <header>
+        <h1> Obi Nnaeto </h1>
+        <div id = "sidebar">
+        <a href = "mailto:obi.nnaeto@gmail.com"><img src = "images/email.png"></a>
+        <a href = "https://www.linkedin.com/in/obi-nnaeto-239933162/"><img src = "images/linkedIn.png"></a>
+        <a href = "https://github.com/cnnaeto21"><img src = "images/git.png"></a>
+        </div>
+        <h3> About Me </h3>
+    </header>
+    <div id = "bio">
+    <p> I am a Sophomore Information Systems Major at Carnegie Mellon University.
+    I am originally from Nigeria and I grew up in Morristown New Jersey I am the middle of 
```

Comment:
nit: comma after city name as such - Morristown, New Jersey. Also needs a period at the end of the sentence.